src/server/api/controllers/create-batch.controller.js

```diff
@@ -1,11 +1,23 @@
 const knex = require('../../config/db');
 const Error = require('../lib/utils/http-error');
 
-const createBatch = async (cropId, userId) => {
+const createBatch = async (
```

Comment:
Check src/server/api/controllers/modules.controller.js as example how to write post method